src/Cpp/1-getting-started/2-HelloFroge/Main.cpp

```diff
@@ -211,8 +188,7 @@ bool Initialize()
     debugInfoQueue->SetBreakOnSeverity(D3D11_MESSAGE_SEVERITY::D3D11_MESSAGE_SEVERITY_CORRUPTION, true);
     debugInfoQueue->SetBreakOnSeverity(D3D11_MESSAGE_SEVERITY::D3D11_MESSAGE_SEVERITY_ERROR, true);
 
-    D3D11_MESSAGE_ID hide[] =
-    {
+    D3D11_MESSAGE_ID hide[] = {
```

Comment:
`{` on a new line again

Reply:
I can't seem to find an option withing clang-format that allows me to break braces in initializer lists... Will keep searching

Reply:
Oki, it might be a named style like knr? i have to look into clang as well